Lib/json/tool.py

```diff
@@ -26,19 +26,26 @@ def main():
                         help='write the output of infile to outfile')
     parser.add_argument('--sort-keys', action='store_true', default=False,
                         help='sort the output of dictionaries alphabetically by key')
+    parser.add_argument('--jsonlines', action='store_true', default=False,
```

Comment:
I suggest to name the option `--json-lines` or just `--lines`.

Reply:
That `--lines` may be more concise.

Reply:
`--json-lines`  named similar to the existing `--sort-keys`,it will be  more suitable.